editor/inserter/menu.js

```diff
@@ -225,7 +225,15 @@ class InserterMenu extends wp.element.Component {
 	render() {
 		const { position = 'top', instanceId } = this.props;
 		const visibleBlocksByCategory = this.getVisibleBlocksByCategory( wp.blocks.getBlockTypes() );
-		const positionClasses = position.split( ' ' ).map( ( pos ) => `is-${ pos }` );
+		const visualEditorHeight = document.querySelector( '.editor-visual-editor' ).clientHeight;
+		const minimumNeededHeight = 600;
```

Comment:
@aduth then maybe if I looked up the height value using getComputedStyles on the .editor-inserter__content element it would make this solution more flexible?